projects/storefrontlib/src/lib/ui/layout/main/storefront.component.html

```diff
@@ -1,4 +1,7 @@
 <cx-header></cx-header>
+
+<cx-dynamic-slot position="BottomHeaderSlot"></cx-dynamic-slot>
```

Comment:
I know it's temporary workaround for sample data. But won't we see the `BottomHeaderSlot` slot twice, when it's also rendered by the `<cx-page-layout>`?

Reply:
Good point. As soon as all pages have transformed to the page layout, we probably move this into configuration and can drop it here. Since this isn't the case yet, we keep it here. Only if the BottomHeaderSlot is added to a layout config, it will show up twice.